code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -449,7 +449,7 @@
 		if((NO_BLOOD in target.dna.species.species_traits) || target.dna.species.exotic_blood || !target.blood_volume)
 			to_chat(user, "<span class='warning'>They have no blood!</span>")
 			return
-		if(target.mind && target.mind.has_antag_datum(/datum/antagonist/vampire))
+		if(target.mind && (target.mind.has_antag_datum(/datum/antagonist/vampire) || target.mind.has_antag_datum(/datum/antagonist/mindslave/thrall)))
```

Comment:
Might be good to note this in the PR description. Or in the CL itself. Pretty big change from what old vamps had.

Reply:
added to the PR description.